File: examples/TeamGetExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TeamApi();
apiCaller.username = "YOUR_API_KEY";
// apiCaller.accessToken = "YOUR_ACCESS_TOKEN";

apiCaller.teamGet().then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling TeamApi#teamGet:");
  console.log(error.body);
});

File: examples/TeamGetExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
    # access_token="YOUR_ACCESS_TOKEN",
)

with ApiClient(configuration) as api_client:
    try:
        response = api.TeamApi(api_client).team_get()

        pprint(response)
    except ApiException as e:
        print("Exception when calling TeamApi#team_get: %s\n" % e)
